packages/component-boilerplate/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/component-boilerplate",
-  "version": "1.5.1",
+  "version": "1.5.1-8",
```

```suggestion
  "version": "1.5.2",
```
newest version needs to be `1.5.2`